### PR TITLE
Update driver.go

### PR DESCRIPTION
**Pull request by olicuzo**
I've stumbled upon the `source driver: unknown driver file (forgotten import?)` error which has been reported as an issue multiple times. 

This is a tiny proposal that might make the error message more readable:
instead of `source driver: unknown driver file (forgotten import?)`, the message can be `source driver: unknown driver 'file' (forgotten import?)` to indicate that 'file' is the type of the driver

### CODE REVIEW

File: source/driver.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func Open(url string) (Driver, error) {
 	d, ok := drivers[u.Scheme]
 	driversMu.RUnlock()
 	if !ok {
-		return nil, fmt.Errorf("source driver: unknown driver %v (forgotten import?)", u.Scheme)
+		return nil, fmt.Errorf("source driver: unknown driver '%v' (forgotten import?)", u.Scheme)
```
Comment (dhui) [inline source/driver.go:90]: Might as well make it `'%s'` while you're at it

**Review by dhui — COMMENTED** [review]
Thanks for the PR!

File: source/driver.go (hunk 1 above)
Comment (olicuzo) [reply source/driver.go]: done

**Review by dhui — APPROVED** [review]
Thanks for the update!

**Pull request merged**